sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/implementation/quickpulse/filtering/TelemetryColumns.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.autoconfigure.implementation.quickpulse.filtering;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TelemetryColumns {
```

Comment:
You could perhaps remove the inheritance (and use composition => loose coupling) and extract the code from https://github.com/Azure/azure-sdk-for-java/blob/c577ae534277be3fda088a805ff528e6048bf116/sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/implementation/quickpulse/filtering/Filter.java#L129 to line 134 to a method of `TelemetryColumns`

Reply:
See https://github.com/Azure/azure-sdk-for-java/pull/43040#discussion_r1854454536